repo/js/ActivitySwitchNotice/main.js

```
@@ -1,7 +1,15 @@
-eval(file.readTextSync(`utils/notice.js`));
-eval(file.readTextSync(`utils/campaignArea.js`));
-
+async function init() {
+    let utils=[
+        "ws",
+        "notice",
+        "campaignArea",
+        "activity",
+    ]
+    for (let util of utils) {
+        eval(file.readTextSync(`utils/${util}.js`));
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)